tests/ui-api-layer-acceptance-tests/graphql/config.go

```diff
@@ -17,9 +16,8 @@ type envConfig struct {
 }
 
 type config struct {
-	GraphQlEndpoint   string
-	LocalClusterHosts []string
-	IdProviderConfig  idProviderConfig
+	GraphQlEndpoint  string
```

Comment:
It should be called `GraphQLEndpoint`, as Golang has specific convention of naming variables with abbreviations etc.